Index: .env
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>DATABASE_NAME=IPPSystem\n\nMYSQL_USERNAME=root\nMYSQL_PASSWORD=123456\nMYSQL_HOST=localhost\nMYSQL_PORT=3306\nMYSQL_URL=jdbc:mysql://localhost:3306/IPPSystem\n\n# ===== SMTP (for OTP email) =====\nSMTP_HOST=smtp.gmail.com\nSMTP_PORT=587\nSMTP_USER=anthtooaung2792005@gmail.com\nSMTP_PASS=stjl lidp hnwc ubus\nSMTP_TLS=true\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.env b/.env
--- a/.env	(revision 4a11405b2171739f5306cda30ebc15304d2470ed)
+++ b/.env	(date 1770109594312)
@@ -1,7 +1,7 @@
 DATABASE_NAME=IPPSystem
 
 MYSQL_USERNAME=root
-MYSQL_PASSWORD=123456
+MYSQL_PASSWORD=
 MYSQL_HOST=localhost
 MYSQL_PORT=3306
 MYSQL_URL=jdbc:mysql://localhost:3306/IPPSystem
